playbooks/cluster/openshift/config.yml

```diff
@@ -89,6 +89,6 @@
     - name: Configure oc admin user for testing
       shell: |
         user_name="{{ admin_test_user_name | default('test_admin') }}"
-        oc login -u system:admin
-        oc get user "$user_name" || oc create user "$user_name"
-        oc adm policy add-cluster-role-to-user cluster-admin "$user_name"
+        oc login -u system:admin --config=/etc/origin/master/admin.kubeconfig
+        oc get user "$user_name" --config=/etc/origin/master/admin.kubeconfig || oc create user "$user_name" --config=/etc/origin/master/admin.kubeconfig
+        oc adm policy add-cluster-role-to-user cluster-admin "$user_name" --config=/etc/origin/master/admin.kubeconfig
```

Comment:
@gouyang Thanks for a contribution. Just two notes: 

* long strings
* what do you think about making `/etc/origin/master/admin.kubeconfig` a variable?

Reply:
I updated the PR to use a variable for this string.